frontend/src/components/Step/Step.spec.tsx

```diff
@@ -2,24 +2,52 @@ import { render, screen } from 'utils/test/test-utils';
 import { Step, StepProps } from './Step';
 import { DropboxIcon } from 'icons/DropboxIcon/DropboxIcon';
 import { expect } from 'vitest';
+import { ThemeProvider } from 'styled-components';
+import { theme } from 'styles/theme';
+import { IconWrapper } from './Step.styles';
 
 const props: StepProps = {
-  icon: <DropboxIcon />,
-  itemStep: 1,
-  title: 'Step 1',
-  currentStep: 1,
-  maxStep: 3,
-  onClick: () => ({}),
+	icon: <DropboxIcon />,
+	itemStep: 1,
+	title: 'Step 1',
+	currentStep: 1,
+	maxStep: 3,
+	onClick: () => ({}),
+	isActive: false,
+	IconWrapper: function (IconWrapper: any): HTMLElement {
+		throw new Error('Function not implemented.');
+	},
 };
 
 describe('<Step />', () => {
-  it('should render the component', () => {
-    render(<Step {...props} />);
+	it('should render the component', () => {
+		render(<Step {...props} />);
 
-    expect(
-      screen.getByText(`Passo ${props.currentStep} / ${props.maxStep}`),
-    ).toBeInTheDocument();
+		expect(
+			screen.getByText(`Passo ${props.currentStep} / ${props.maxStep}`),
+		).toBeInTheDocument();
 
-    expect(screen.getByText(props.title)).toBeInTheDocument()
-  });
+		expect(screen.getByText(props.title)).toBeInTheDocument();
+	});
+	it('should have active icon when currentStep is greater than or equal to itemStep', () => {
```

Comment:
```suggestion
	it('should have active icon when currentStep is greater than itemStep', () => {
```

Separa e faz um teste qundo é igual